src/terragraph-e2e/e2e/minion/BgpUtils.cpp

```diff
@@ -153,11 +148,9 @@ BgpUtils::fetchVtyshBgpStatus() {
     return bgpStatus;
   }
   std::string summaryCommand, advertisedRoutesFormat, receivedRoutesFormat;
-  if (versionOutput.value().rfind("Quagga", 0) == 0) {
-    summaryCommand = kVtyshQuaggaGetBgpSummaryCommand;
-    advertisedRoutesFormat = kVtyshQuaggaGetBgpAdvertisedRoutesFormat;
-    receivedRoutesFormat = kVtyshQuaggaGetBgpReceivedRoutesFormat;
-  } else /* if (versionOutput.value().rfind("FRRouting", 0) == 0) */ {
+ 
+  //FRROUTING
```

Comment:
a lot of these new comments are unnecessary... please delete them

Reply:
Addressed the review comments